internal/distro/rhel90/distro.go

```diff
@@ -460,12 +460,34 @@ func (t *imageType) checkOptions(customizations *blueprint.Customizations, optio
 		}
 
 		if t.name == "edge-simplified-installer" {
-			if err := customizations.CheckAllowed("InstallationDevice"); err != nil {
+			if err := customizations.CheckAllowed("InstallationDevice", "FDO"); err != nil {
 				return fmt.Errorf("boot ISO image type %q contains unsupported blueprint customizations: %v", t.name, err)
 			}
 			if customizations.GetInstallationDevice() == "" {
 				return fmt.Errorf("boot ISO image type %q requires specifying an installation device to install to", t.name)
 			}
+			if customizations.GetFDO() == nil {
+				return fmt.Errorf("boot ISO image type %q requires specifying FDO configuration to install to", t.name)
+			}
```

Comment:
we need a test plan that:

- configures and brings up a manufacturing server (via container perhaps)
- specify the FDO manufacturing server url in the blueprint
- specify either insecure or hash to verify the connection (test both)
- test installation

Reply:
probably better to validate the full flow as per my comment below

Reply:
Yi is testing this case with insecure - we can evolve the test for hash check later on